reference/settings.v3.yml

```diff
@@ -924,6 +924,21 @@ components:
         store_country:
           type: string
           default: United States
+        shopper_language_selection_method:
+          type: string
+          enum:
+            - browser
+            - default_shopper_language
+          example: browser
+          default: browser
```

Comment:
default value would be a `default_shopper_language`

Reply:
I guess example would also be better to provide for `default_shopper_language` selected as a default value for `shopper_language_selection_method`